invenio_vocabularies/contrib/affiliations/jsonschemas/affiliations/affilation-v1.0.0.json

```diff
@@ -5,49 +5,28 @@
   "additionalProperties": false,
```

Comment:
there is a typo in the name of the file, it should be:

`affilation-v1.0.0.json` -> `affiliation-v1.0.0.json`